src/api/scope/lib/resolver.js

```diff
@@ -9,7 +9,7 @@ export const setResolver = (currentPath: string, resolverPath: string): Promise<
 };
 
 export const getResolver = (currentPath: string): Promise<string> => {
-  return loadScope(currentPath).then(scope => scope.scopeJson.resolverPath);
+  return loadScope(currentPath).then(scope => scope.scopeJson.resolverPath || '');
```

Comment:
Not sure we expect an empty string when there is no resolvePath.
I would leave it and change the `: Promise<string>` to `: Promise<?string>` to indicate `string` or `null/undefined`.

Reply:
This creates a ripple effect, right !? Because the places that use of function don't expect to get a `?string`. Even though the code should support it. This creates more errors. How do I know this? Did loads of other fixes last night, but before I committed  I looked at the ESlint report and found out that the number of errors increased. 

If I do find all references and fix in the consuming functions, I create even more ripple effects... because the function call tree increases.  ~sigh~.

Reply:
checked this isn't this case.

Reply:
I see what you're saying, it happens often when fixing Flow errors, but it's a good thing because as a result, you fix many other places as well.

To me also it makes the process of fixing Flow errors more tedious.